### ISSUE TITLE
Please implement Text shadow feature, especially in Presentation

### ISSUE THREAD

**Issue by mriza** [ONLYOFFICE/DesktopEditors#389]
To increase the contrast between text and background with similar colors, one can apply shadows to object or foreground objects.  #unfortunately, this feature is missing in Presentation. Please implement it. Thanks

**Comment by Rita-Bubnova** [comment]
Hello, @mriza.
Shadow apply with shapes and images (tab "Shape settings" right panel properties -> check box "Show shadow" ).
But shadow incorrect work with text (placeholder) - this is know issue 42321 in our private issue tracker.

Problems reproduces in Document Server.

**Comment by mriza** [comment]
yes, that's what I reproduced in Desktop Editor too. I've also test text shadow interoperability with WPS Office and Libreoffice too. Both software also failed to show text shadow from document that saved using MS Office.

**Comment by higheraims** [comment]
I've checked OO version 8 to see if the behavior has changed. Still an interoperability issue - soft edged drop shadows on text gets stripped out sadly.

**Comment by LClayville** [comment]
Is this still being worked on? I'm working on a presentation and I can't seem to find a proper way to add a shadow to a text box.

**Comment by higheraims** [comment]
I confirm it's still a problem in version 8.2. Presentations don't support drop shadows. Keep in mind we really want soft edged drop shadows here. That's what the closed-source presentation apps support.
